src/Stryker.Core/Stryker.Core.UnitTest/MutantFilters/FilePatternMutantFilterTests.cs

```diff
@@ -16,7 +15,7 @@ public class FilePatternMutantFilterTests : TestBase
         [Fact]
         public static void ShouldHaveName()
         {
-            var target = new FilePatternMutantFilter(new StrykerOptions()) as IMutantFilter;
```

Comment:
Dunno why this was there. Anyway, nice that options are not need for showing display name.